.github/workflows/test.yml

```diff
@@ -30,15 +30,13 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
         id: go
-      - name: Set up GOPATH
+      - name: Set up INSTALL_DIRECTORY
         run: |
-          go version
-          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
-          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+
           mkdir -p "bin"
           mkdir -p "pkg"
           mkdir -p "src"
```

Comment:
Maybe for another PR but it looks like these should be handled by the makefile (well I can't imagine test-with-race to need any folders lke this set up)

Reply:
Thanks for the call out! You're absolutely right, I removed these steps and ran them on my fork. I found out that [the tests pass just fine without them](https://github.com/NathanielRN/aws-xray-sdk-go/actions/runs/1719812674).

I thought about making another PR but since this is just deleting lines I thought it would be fine to put them in the same PR. I'll just update the PR title!